node/consensus/consensus.go

```diff
@@ -600,7 +611,14 @@ func (c *Consensus) verifyCE(req []byte) (smartbft_types.RequestInfo, *state.Con
 	} else if ce.BAF != nil {
 		return reqID, ce, c.SigVerifier.VerifySignature(ce.BAF.Signer(), ce.BAF.Shard(), toBeSignedBAF(ce.BAF), ce.BAF.Signature())
 	} else if ce.ConfigRequest != nil {
-		return reqID, ce, nil // TODO verify sig over config
+		_, err := c.verifyAndClassifyRequest(&protos.Request{
+			Payload:   ce.ConfigRequest.Envelope.Payload,
+			Signature: ce.ConfigRequest.Envelope.Signature,
+		})
+		if err != nil {
+			return reqID, ce, errors.Wrapf(err, "failed to verify and classify request")
+		}
+		return smartbft_types.RequestInfo{}, nil, nil
```

Comment:
This is an empty return.... if it is in preparation for the next stage put a TODO with an explanation

Reply:
please leave the TODO but in the meantime return reqID, ce, nil